### DISCUSSION TITLE
Autoplay easing function with Swiper WebComponents

### DISCUSSION THREAD

**Discussion by fabien-lg** [nolimits4web/swiper#6948] [Q&A] [answered]
Hello,

I want to have an infinite, smooth autoplay. I've seen that many users have succeeded by adding this CSS on `.swiper-wrapper`.
```
transition-timing-function: linear; 
```

But in the [recent documentation](https://swiperjs.com/element), `swiper-wrapper` no longer exists. 

How can I achieve the same result?

Thanks!

**Comment by Arhell** [comment]
Try https://open-code.tech/en/post-728/

**Reply by fabien-lg** [reply, marked as answer]
@Arhell Thanks for the link!

In the meantime, we found a solution. The `swiper-wrapper` is injected in a Shadow DOM, so we adjusted the `injectStyles` property:
```
injectStyles: [
     '.swiper-wrapper { transition-timing-function: linear !important }',
],
```
